game/core/src/me/lihq/game/screen/elements/OrthogonalTiledMapRendererWithSprite.java

```diff
@@ -59,12 +91,19 @@ public void render()
             renderTileLayer((TiledMapTileLayer) layer);
 
             if (currentLayer == amountOfLayers - 2 || amountOfLayers == 1) {
-                for (Sprite s : sprites) {
+
+                for (Sprite s : people)
+                {
                     s.draw(this.getBatch());
                 }
             }
         }
 
+        if (Settings.DEBUG)
```

Comment:
maybe move this into the nav screen with the other render debug method (you could add a debug.render(map, batch) that calls both render methods?